fido2-demo/base/src/main/java/com/linecorp/line/auth/fido/fido2/base/service/UserKeyServiceImpl.java

```diff
@@ -74,7 +75,7 @@ public UserKey createUser(UserKey user) {
     public boolean isRegistered(String rpId, String userId) {
         List<UserKeyEntity> userKeyEntities = userKeyRepository.findAllByRpEntityIdAndUserId(rpId, userId);
         return userKeyEntities != null &&
-               !userKeyEntities.isEmpty();
+                !userKeyEntities.isEmpty();
```

Comment:
How about writing this code with the 77 line together?

Reply:
Changes completed. Thank you for your help.